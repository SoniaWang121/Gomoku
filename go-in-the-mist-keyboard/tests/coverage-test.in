who
B
place Z1
who
B
place A1
who
W
place A1
who
W
place B122
who
W
place B1
who
B
place A2
place B2
place A3
place B3
place A4
place B4
place A5