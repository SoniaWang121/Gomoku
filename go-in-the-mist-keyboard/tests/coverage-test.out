who
B
place Z1
Invalid coordinate
who
B
place A1
who
W
place A1
Occupied coordinate
who
W
place B122
Invalid coordinate
who
W
place B1
who
B
place A2
place B2
place A3
place B3
place A4
place B4
place A5
Black wins!
A1B1A2B2A3B3A4B4A5
Thank you for playing!